hail/python/test/hail/methods/test_statgen.py

```diff
@@ -1278,6 +1278,16 @@ def test_split_multi_table(self):
         ds1 = ds1.drop('was_split', 'a_index')
         self.assertTrue(ds1._same(ds2))
 
+    def test_split_multi_shuffle(self):
+        ht = hl.utils.range_table(1)
+        ht = ht.annotate(keys=[hl.struct(locus=hl.locus('1', 1180), alleles=['A', 'T']),
+                               hl.struct(locus=hl.locus('1', 1180), alleles=['A', 'C', 'G'])])
```

Comment:
I don't think this will do the thing you want it to do; the exploded, re-keyed table you're passing in to split_multi will already look like
```
1:1180 A,C,G
1:1180 A,T
```
which is already sorted. Maybe something like 
```
1:1180 A,C,T
1:1180 A,G
```
would work?
(and maybe collect the unsplit alleles after rekeying and assert that they're going to split into the out-of-order thing you expect?)

Reply:
huh, it totally looks like you should be right, but this was failing before I fixed the split multi flag passing. I'll investigate.